### PR TITLE
okr: adding priorities and graphsync

### PR DESCRIPTION
**Pull request by mikeal**
@vmx @warpfork please look these over and make sure you have enough context to complete the implementations this quarter.

/cc @mmomack2 @jbenet @Stebalien @whyrusleeping

### CODE REVIEW

File: okrs/2019-q1.md [hunk 1]
```diff
@@ -1,27 +1,40 @@
+## Objective: IPLD can efficiently replicate blockchain data.
+
+Background:
+
+* [WIP Selector Specification](https://github.com/ipld/specs/pull/74/files)
+
+| Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
+| ---------- | -------- | -------- | ----------- | --------------- | ----------- |
+| IPLD Selector grammar can be parsed into an AST w/ support for basic grammar in Go. | @warpfork | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| IPLD Selector grammar can be parsed into an AST w/ support for basic grammar in JS. | @vmx | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
```
Comment (vmx) [inline okrs/2019-q1.md:10]: I'm not sure if we should make parsing a goal. I think as a first step I'd just agree on an AST (JS and Go need to understand it) and implement that. Parsing from a string can be done later on.

I expect the AST to be CBOR, so for JS you'd just manually create some JSON and make CBOR out of it.

File: okrs/2019-q1.md [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
+| Standalone JS implementation of `unixfsv2` is path based.| @mikeal | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg) | | | |
 
 ## Objective: Replication of IPLD graphs is modularized to enable development of new replicators.
 
-| Key Result | Assignee | Mid-Q Score | Projected Score | Final Score |
-| ---------- | -------- | ----------- | --------------- | ----------- |
-| Demo of IPLD replication without Bitswap.| @vmx | | | |
+| Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
+| ---------- | -------- | -------- | ----------- | --------------- | ----------- |
+| Demo of IPLD replication without Bitswap.| @vmx | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg) | | | |
```
Comment (vmx) [inline okrs/2019-q1.md]: I'd remove that one, given that the transport stuff is likely to happen on the Go side of things which will then later be implemented in JS.

File: okrs/2019-q1.md (hunk 1 above)
Comment (mikeal) [reply okrs/2019-q1.md:10]: I'll be much more confident that the AST definition is correct if we have a parser that generates it ;)

Comment (vmx) [reply okrs/2019-q1.md:10]: That's fine if we keep it to "some syntax" and not bikeshed on it. The syntax can be changed later on, it's not a priority to find the ideal on in this quarter.

**Review by vmx — APPROVED** [review]

**Pull request merged**